release-notes/v1_6.md

```diff
@@ -7,70 +7,88 @@ MetaDescription: See what is new in the Visual Studio Code September 2016 Releas
-* **Editor**: Format On Save, Vim style relative line numbers.
-* **Languages**: TypeScript 2.0 language features which benefits both TypeScript and JavaScript development.
-* **Extensions**: Create workspace recommended Extension lists. Extension packs to automatically include dependencies.
-* **Debugging**: Support for running NPM and other script runners on launch. 
-* **Extension Authoring**: New APIs to update settings and modify files before save. Debug protocol updates including programmatic access to `launch.json`.
+![1.6 Release hightlights include: TypeScript 2.0, Release notes within VS Code, Format on Save, Swich between Windows, Search history, run scripts on debugger launch, workspace extension recomendations, extensiosn packs, Extensions updates for JS/TS colorizors, VIM, Experimental Node Debugger and a new API for settings access](/release-notes/images/1_6/ReleaseHighlights_1.6.png)
+
+Ok lets move onto some fo the specific items...
 
 ## Workbench
 
+### Release notes inside VS Code
+
+There is no need to go to our website to review the latest release notes - you can now see these inline within VS Code itself.  Use the global **Show Release Notes** command or the **Help** > **Release Notes** menu to access them.  With any luck this is how you are reading these notes right now :)
+
+
+### Quickly switch between running VS Code instances
+
+We added a new command to let you quickly move between open instances of VS Code.  Simply open the command pallette **TODO add Command** and type **Switch Window**, a picker showing all the opened VS Code Windows (instances) will pop up and you can quickly move from one to the other without leaving the keyboard and right from within VS Code.
+
+
 ### Icons Everywhere
 
 Continuing on the work we did in the August release to support file icons, we've added file icons to many more places throughout the UI. This includes most trees that show files (for example, Problems panel, Search results, OPEN EDITORS section) as well as in **Quick Open** results and the editor headings.
 
-A new setting `workbench.editor.showIcons` controls whether icons should show up for editors.
+If you don't want to see icons everywhere for instance in Tab headings.  We added a new setting `workbench.editor.showIcons`.
 
 ![Icons](images/1_6/icons.png)
 
-### Electron update
 
-In this release, we decided to take a major update of the [Electron](https://github.com/electron/electron) framework. We now run with version **1.3.7** which brings big version increments to the Chrome rendering engine (from **49** to **52**) and Node.js (from **5.10.0** to **6.5.0**). This also means that all extensions can expect the full Node.js **6.5.0** runtime when running in this VS Code release. For a full list of changes in Node.js, please refer to the [Node.js Changelog](https://github.com/nodejs/node/blob/master/CHANGELOG.md).
+### Detecting a corrupt installation
 
-![Icons](images/1_6/electron.png)
```

Comment:
Agree.  I tried to crop the bottom but was having trouble on Windows with Paint.  The outer transparency was turning to a white background.  Can you crop it easily on your Mac.